CONTRIBUTING.md

```diff
@@ -115,15 +115,15 @@ log is formatted as a short title (first line) and an extended description
 The short title is the most important part, as it is what will appear in the
 `shortlog` changelog (one line per commit, so no description shown) or in the
 GitHub interface unless you click the "expand" button. As the name tells it,
-try to keep that first line relatively short (ideally <= 50 chars, though it's
+try to keep that first line relatively short (ideally <= 50 characters, though it's
 rare to be able to tell enough in so few characters, so you can go a bit
 higher) - it should describe what the commit does globally, while details would
 go in the description. Typically, if you can't keep the title short because you
 have too much stuff to mention, it means that you should probably split your
 changes in several commits :)
 
 Here's an example of a well-formatted commit log (note how the extended
-description is also manually wrapped at 80 chars for readability):
+description is also manually wrapped in 80 chars for readability):
```

Comment:
As far as I know "at" is correct here. The action is "to wrap", and it happens at the threshold of 80 characters. We're not wrapping the line "in" 80 characters, which IMO does not make sense (80 characters is not a packaging).